.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,22 @@
+## For issue
+
 ## Please verify the following:
 
 - [ ] `yarn test` **jest** tests pass with new tests, if relevant
 - [ ] `yarn lint` **eslint** checks pass with new code, if relevant
 - [ ] `yarn format:check` **prettier** checks pass with new code, if relevant
 - [ ] `README.md` (or relevant documentation) has been updated with your changes
+- [ ] Include issue number in title and description
+- [ ] (If applicable) The feature related to this PR was tested across various use cases to ensure it works correctly.
 
 ## Describe your PR
+
+[Add your PR description]
+
+## Screenshots (if applicable)
+
+If you’re submitting code changes related to a visible feature, please include before-and-after screenshots or videos. This helps the reviewer easily compare and understand the changes.
```

Comment:
```suggestion
If you’re submitting code changes related to a visible feature, please include before-and-after screenshots or videos.
```

I prefer this shorter explanation because:

1. Simple instructions
2. even with screenshots/videos, I prefer reviewers also run the code themselves on non-trivial changes. the screenshots/videos are also to help make sure the implementer tested it out.